datafusion/physical-expr/src/datetime_expressions.rs

```diff
@@ -423,6 +457,11 @@ fn to_timestamp_impl<T: ArrowTimestampType + ScalarType<i64>>(
     }
 }
 
+/// to_date SQL function
+pub fn to_date(args: &[ColumnarValue]) -> Result<ColumnarValue> {
```

Comment:
Can we add some rustdoc with a dataframe based example to this? See [this as an example](https://github.com/apache/arrow-datafusion/blob/main/datafusion/physical-expr/src/regex_expressions.rs#L99)